src/index/views/action-Send.vue

```diff
@@ -0,0 +1,392 @@
+  },
+
+  mounted() {
+    this.loadAccount(this.$store.state.currentIndexAccount);
+  },
+
+  methods: {
+    setMaxAmount() {
+      this.amount = this.balance;
+    },
+    toggleAdvanced() {
+      this.showAdvanced = !this.showAdvanced;
+    },
+    changeTo() {
+      this.isToValidated = false;
+      this.resolvedKap = "";
+    },
+    async validateTo() {
+      this.isToValidating = true;
+      let isAddress;
+      try {
+        isAddress = utils.isChecksumAddress(this.to);
+      } catch (_) {
+        isAddress = false;
+      }
+      if (isAddress) {
+        this.isToValid = true;
+      } else if (this.network.kapNameServiceContractId) {
+        try {
+          const buffer = new TextEncoder().encode(this.to);
+          const kapHex = `0x${utils.toHexString(buffer)}`;
+          const { result } = await this.kapNameService.owner_of({
+            token_id: kapHex,
+          })
+          this.resolvedKap = result?.value;
+          this.isToValid = !!this.resolvedKap;
+        } catch (_) {
+          this.isToValid = false;
+        }
+      } else {
+        this.isToValid = false;
+      }
+      this.isToValidating = false;
+      this.isToValidated = true;
+    },
+    validateAmount() {
+      const amount = parseFloat(this.amount);
+      this.isAmountValid =
+        !isNaN(this.amount) &&
+        !isNaN(amount) &&
+        0 <= amount &&
+        this.balance >= amount;
+    },
+    async loadAccount(index) {
+      if (this.$store.state.accounts.length === 0) return;
+      try {
+        this.$store.state.networks = await this._getNetworks();
+        const currentTag = await this._getCurrentNetwork();
+        this.$store.state.currentNetwork = this.$store.state.networks.findIndex(
+          (n) => n.tag === currentTag
+        );
+        this.network =
+          this.$store.state.networks[this.$store.state.currentNetwork];
+        this.provider = new Provider(this.network.rpcNodes);
+        const currentAccount = this.$store.state.accounts[index];
+        this.address = currentAccount.address;
+        this.signer = undefined;
+        if (currentAccount.privateKey) {
+          this.signer = Signer.fromWif(currentAccount.privateKey, true);
+          this.signer.provider = this.provider;
+        } else {
+          router.back();
+        }
+
+        this.koinContract = new Contract({
+          id: this.network.koinContractId,
+          abi: utils.tokenAbi,
+          provider: this.provider,
+          signer: this.signer,
+          serializer: await this.newSandboxSerializer(
+            utils.tokenAbi.koilib_types
+          ),
+        });
+        this.koin = this.koinContract.functions;
+
+        if (this.network.kapNameServiceContractId) {
+          const kapAbi = await this._getAbi(this.network.tag, this.network.kapNameServiceContractId);
+          this.kapNameServiceContract = new Contract({
+            id: this.network.kapNameServiceContractId,
```

Comment:
Right now I have problems to resolve kap names. Further investigation reveal this error
```
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".
```

To solve this use the serializer from the sandbox:
```
          this.kapNameServiceContract = new Contract({
            id: this.network.kapNameServiceContractId,
            abi: kapAbi,
            provider: this.provider,
            serializer: await this.newSandboxSerializer(
              kapAbi.koilib_types
            ),
          });
```